dealership/vehicles.py

```diff
@@ -1,15 +1,64 @@
 class Vehicle(object):
     def __init__(self, maker, model, year, base_price, miles):
-        pass
+        self.maker = maker
+        self.model = model
+        self.year = year
+        self.base_price = base_price
+        self.miles = miles
 
 
 class Car(Vehicle):
-    pass
+    # Define purchase multipliers
+    def __init__(self, maker, model, year, base_price, miles):
+        # Inherit vehicle attributes
+        super(Car, self).__init__(maker, model, year, base_price, miles)
+    
+    sale_multiplier = 1.2
```

Comment:
Constants are generally all uppercase, something like `SALE_MULTIPLIER`. Not an issue, just convention style.